uint8 work_mode  # 0 - MANUAL, 1 - AUTO
uint8 nozzle_type  # 0 - NONE, 1 - BRUSH, 2 - EMA
uint8 lock_status  # 0 - LOCKED, 1 - UNLOCKED
---
bool success
